src/components/packages/PackagesContainer.js

```diff
@@ -94,6 +112,21 @@ class PackagesContainer extends Component {
                     cardMinWidth                = 315;
                     maxMobileScreenWidth        = 768;
                     mobileSectionOffset         = 20;
+                    
```

Comment:
@xiahengchen I would like to know the reasoning for moving this logic from `adjustPackages` to `init`. As far as I can tell, during the first render it will not make any changes. But since `adjustPackages` is the one listening to `window.resize`, this logic will not be executed anymore.